### ISSUE TITLE
Loading issue with modules that shouldn't be generated: AWS.Cloudsearchdomain and AWS.Importexport (Instead of CamelCase counterparts)

### ISSUE THREAD

**Issue by iguberman** [aws-beam/aws-elixir#148]
Problem with either  0.13.0"  or 0.12.0.   Been using this gependency for local dev/testing no problem.   But when I run `mix release` and consequently path/to/rel/my_app start |start_iex|daemon|etc.:
>path/to/rel/my_app start 
(no logger present) unexpected logger message: {log,error,"~s~n",["beam/beam_load.c(180): Error loading module 'Elixir.AWS.CloudSearchDomain':\n  module name in object code is 'Elixir.AWS.Cloudsearchdomain'\n"],#{error_logger=>#{emulator=>true,tag=>error},gl=><0.0.0>,pid=><0.651.0>,time=>1662242247916457}}
(no logger present) unexpected logger message: {log,error,"~s~n",["beam/beam_load.c(180): Error loading module 'Elixir.AWS.Importexport':\n  module name in object code is 'Elixir.AWS.ImportExport'\n"],#{error_logger=>#{emulator=>true,tag=>error},gl=><0.0.0>,pid=><0.742.0>,time=>1662242247971255}}
{"init terminating in do_boot",{load_failed,['Elixir.AWS.Importexport','Elixir.AWS.CloudSearchDomain']}}
init terminating in do_boot ({load_failed,[Elixir.AWS.Importexport,Elixir.AWS.CloudSearchDomain]})

Crash dump is being written to: erl_crash.dump...done
>grep Slogan erl_crash.dump
Slogan: init terminating in do_boot ({load_failed,[Elixir.AWS.Importexport,Elixir.AWS.CloudSearchDomain]})

Things in `deps/lib/aws/generated` dir  look right:
`cloudsearchdomain.ex` starts with `defmodule AWS.Cloudsearchdomain do` and `cloud_search_domain.ex` ` defmodule AWS.CloudSearchDomain do`    
But  `AWS.CloudSearchDomain.beam` in the release dir indeed has `defmodule AWS.Cloudsearchdomain do`
Same story with `AWS.ImportExport.beam` -- `defmodule AWS.Importexport do`
At the same time similar module AWS.Codestarnotifications.beam with defmodule AWS.Codestarnotifications do is no problem.

These "twin" modules look the same  to me after  looking briefly, so only CamelCase ones should get generated. 

```
>iex
Erlang/OTP 25 [erts-13.0.2] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [jit:ns] [dtrace]
Interactive Elixir (1.13.4) - press Ctrl+C to exit (type h() ENTER for help)
```

**Comment by philss** [comment]
Hi, @iguberman :wave: 

Thanks for reporting this! I think it's fixed now. I released a new version, `v0.13.1`. Please update.

**Comment by iguberman** [comment]
Fantastic!   Thank you so much! @philss  I'll update.
